doc/input-output-reference/src/overview/group-unitary-equipment.tex

```diff
@@ -718,27 +723,75 @@ \subsubsection{Inputs}\label{inputs-1-046}
 
 \paragraph{Field: Heating Speed 2 Supply Air Flow Ratio}\label{field-heating-speed-2-supply-air-flow-ratio}
 
-This numeric field defines the ratio of supply air flow rate leaving the unitary system to the maximum air flow rate specified in the coil object at maximum speed when the heating coil and/or supplemental heater are operating at Speed 2. Values must be greater than 0. The entered value must be greater or equal to the flow rate ratio specified for heating speed 1.
+This numeric field defines the ratio of supply air flow rate leaving the unitary system to the maximum air flow rate specified in the coil object at maximum speed when the heating coil and/or supplemental heater are operating at Speed 2. Values must be greater than 0. The entered value must be greater or equal to the flow rate ratio specified for heating speed 1. If the `Number of Speeds for Heating' is less than 2, then this field can be left blank.
```

Comment:
I think I recall that I committed this at work but didn't push since I was waiting for other comments.